buildSrc/src/main/kotlin/project/base.gradle.kts

```diff
@@ -13,3 +13,17 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+package project
+
+plugins { id("conventions.git-version") }
+
+group = "org.mongodb"
+
+// TODO update Github publish action.
```

Comment:
Reminder to remove this TODO after updating the GH action.  Can it be done as part of this PR?

Reply:
I don't know how the action works - will look.

Reply:
Done 👍